megatron/arguments.py

```diff
@@ -251,6 +251,11 @@ def parse_args(extra_args_provider=None, defaults={},
 
     args.curriculum_learning = False
 
+    # Activation function
+    if args.glu_activation is not None:
+        assert not args.bias_gelu_fusion, \
+            'if glu activation is used, please set --no-bias-gelu-fusion'
```

Comment:
```suggestion
    if args.glu_activation is not None and args.bias_gelu_fusion:
        raise ValueError("if glu-activation is used, please set --no-bias-gelu-fusion")
```

Let's gradually transition to the above syntax from asserts. asserts are best used in the testing env.